src/api/routers/participants/participantsCreation.ts

```diff
@@ -3,32 +3,29 @@ import { z } from 'zod';
 
 import { ApiRole } from '../../entities/ApiRole';
 import {
-  Participant,
-  ParticipantApprovalPartial,
-  ParticipantStatus,
+  Participant, ParticipantStatus
 } from '../../entities/Participant';
-import { ParticipantType } from '../../entities/ParticipantType';
 import { User, UserCreationPartial } from '../../entities/User';
 import { getKcAdminClient } from '../../keycloakAdminClient';
 import { addSite, getSiteList, setSiteClientTypes } from '../../services/adminServiceClient';
 import {
   mapClientTypesToAdminEnums,
-  SiteCreationRequest,
+  SiteCreationRequest
```

Comment:
I think lint-wise, this should have a comma like in the original - it gets auto corrected to a comma when I save it on my machine

Reply:
nice pickup. I think my auto-formatting broke - restarting VS Code seemed to fix them all.